bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/DiscoveryServiceRegistry.java

```diff
@@ -21,6 +21,7 @@
  * </ul>
  *
  * @author Michael Grammling - Initial Contribution.
+ * @author Ivaylo Ivanov <ivivanov.bg@gmail.com> - Added getMaxScanTimeout
```

Comment:
please remove your e-mail from the author tags (to be in sync with the other entries)